simphony/io/tests/test_h5_lattice.py

```diff
@@ -38,8 +39,10 @@ def cleanup(self):
         shutil.rmtree(self.temp_dir)
 
     def container_factory(self, name, type_, base_vect, size, origin):
+        self.group = self.handle.create_group(
```

Comment:
For the tests it does not look that there is a reason to create another group node.